omero/sysadmins/version-requirements.rst

```diff
@@ -164,38 +137,23 @@ Linux (Ubuntu)
     * - Version
       - Release date
       - Supported until
-      - OMERO 5.4
-      - OMERO 5.5
       - OMERO 5.6
-    * - 14.04 LTS
-      - Apr 2014
-      - Apr 2019
-      - |Supported|
-      - |Dropped|
-      - |Dropped|
-    * - 16.04 LTS
-      - Apr 2016
-      - Apr 2021
-      - |Recommended|
-      - |Recommended|
-      - |Dropped|
     * - 18.04 LTS
       - Apr 2018
       - Apr 2028
-      - |Upcoming|
       - |Supported|
-      - |Recommended|
     * - 20.04 LTS
       - Apr 2020
       - Apr 2030
-      - |Upcoming|
-      - |Upcoming|
-      - |Upcoming|
+      - |Supported|
```

Comment:
Should this be `|Deprecated|` as per  https://www.openmicroscopy.org/2023/07/24/linux-distributions.html  ?